zerver/lib/integrations.py

```diff
@@ -378,7 +378,13 @@ def __init__(self, name: str, *args: Any, **kwargs: Any) -> None:
 ]
 
 WEBHOOK_INTEGRATIONS: list[WebhookIntegration] = [
-    WebhookIntegration("airbrake", ["monitoring"]),
+    WebhookIntegration(
+        "airbrake",
+        ["monitoring"],
+        config_options=[
+            WebhookConfigOption(name="mapping", description="", validator=check_string)
```

Comment:
I wonder if we want to start adding proper documentation for the special config options? especially if the special settings have generic names. 

I imagine a config option like "mapping" isn't something integrations are unlikely to use, which isn't the worst scenario to happen as that can be rolled back or corrected easily, but it'd be one more unwritten rule to keep in mind when making changes to the integrations.

Reply:
I guess one practical option is probably to store this in maybe, `WebhookConfigOption` as a class variable and have a brief comment there for context around these settings. So here, we use them like this:

```suggestion
            WebhookConfigOption(name=WebhookConfigOption.special_setting, description="", validator=check_string)
```

Could be its own Class or maybe an Enum, but the point is still the same.